code/src/java/pcgen/io/exporttoken/WeaponToken.java

```diff
@@ -1947,9 +1948,10 @@ else if (hitMode == HITMODE_OHHIT)
 		 but for some reason they can (e.g. Monkey Grip) then check
 		 for TOHIT modifiers */
 
-		if (eq.getLocation() == EquipmentLocation.EQUIPPED_PRIMARY
-			|| eq.getLocation() == EquipmentLocation.EQUIPPED_SECONDARY
-			|| eq.getLocation() == EquipmentLocation.EQUIPPED_TWO_HANDS)
+		if (Stream.of(EquipmentLocation.EQUIPPED_PRIMARY,
+				EquipmentLocation.EQUIPPED_SECONDARY,
+				EquipmentLocation.EQUIPPED_TWO_HANDS)
+		          .anyMatch(equipmentLocation -> eq.getLocation() == equipmentLocation))
```

Comment:
What do these equipped locations share?  See below for having this semantically useful